app/components/admin/AdminDashboard.tsx

```diff
@@ -158,7 +163,44 @@ export default function AdminDashboard({ products: newProducts }: Props) {
                   <Td>{product.price}</Td>
                   <Td>
                     {!product.isArchived && (
-                      <Flex justify="center" alignItems="center" height="100%">
+                      <Menu>
+                        <MenuButton
+                          as={Button}
+                          rounded={'full'}
+                          variant={'link'}
+                          cursor={'pointer'}
+                          minW={0}
+                        >
+                          <Icon color="black">
+                            <BsThreeDots />
+                          </Icon>
+                        </MenuButton>
+                        <MenuList>
+                          <MenuItem as="a" href={`product/${product.id}`}>
+                            <Button colorScheme="green">Edit</Button>
+                          </MenuItem>
+                          <MenuItem
+                            as="div"
+                            onClick={() => handleDeleteClick(product)}
+                          >
+                            <Button colorScheme="red">Delete</Button>
+                          </MenuItem>
+                        </MenuList>
+                      </Menu>
+                    )}
+                  </Td>
+                </Tr>
+              );
+            })}
+          </Tbody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+}
+
+{
+  /* <Flex justify="center" alignItems="center" height="100%">
```

Comment:
u could remove the commented code if we are not using it

Reply:
thanks you! I have removed them now